nilearn/datasets/struct.py

```diff
@@ -19,6 +19,10 @@
 # Useful for the very simple examples
 MNI152_FILE_PATH = os.path.join(_package_directory, "data",
                                 "avg152T1_brain.nii.gz")
```

Comment:
I think we just have to update this file and this should be good! :tada:

Reply:
@NicolasGensollen thanks for all feedback and my apologies for the lag. I am trying now to update the whole-brain mask, as you suggested. What I did was to resample this file using as target_affine the gm mask. However, when I run now the pytest locally I get this error:

```python
def test_compute_brain_mask():
        image = Nifti1Image(np.ones((9, 9, 9)), np.eye(4))
    
        mask = compute_brain_mask(image, threshold=-1)
        mask1 = np.zeros((9, 9, 9))
        mask1[2:-2, 2:-2, 2:-2] = 1
    
        np.testing.assert_array_equal(mask1, get_data(mask))
    
        # Check that we get a useful warning for empty masks
        with pytest.warns(masking.MaskWarning):
>           compute_brain_mask(image, threshold=1)
E           Failed: DID NOT WARN. No warnings of type (<class 'nilearn.masking.MaskWarning'>,) was emitted. The list of emitted warnings is: [UserWarning('Resampling binary images with continuous or linear interpolation. This might lead to unexpected results. You might consider using nearest interpolation instead.')].

nilearn/tests/test_masking.py:203: Failed

```

What do you recommend? Thx once again!

Reply:
>What I did was to resample this file using as target_affine the gm mask

@alpinho isn't there a way to download the new template rather than manually resampling the one we currently have? (sorry if this is a stupid question..)

Concerning the test failure: This test is just checking that a warning is given if the mask is empty here:

https://github.com/nilearn/nilearn/blob/c39a82b3d2f926df6bafbf0016b9fd273f06d9ba/nilearn/masking.py#L629-L636
 
It could be that the resampling you did on the template changed the result of:

```python
mask = get_data(resampled_template) >= threshold
```

to be non-empty, which doesn't produce a warning and make the test fail. 

Note that updating the template to a better resolution might break some tests (actually this would be reassuring...), and we will have to update them as well.

Reply:
I downloaded another whole-brain template, called `mni_icbm152_t1_tal_nlin_sym_09a.nii.gz` from [here](https://osf.io/7pj92/download). It has the same resolution as the gm and wm masks but it contains also the skull. It can be removed with FSL BET but I wonder whether there's a function in nilearn that imports this FSL function, so that it can be done quickly in python.

Reply:
Btw, I forgot to mention that in the same link we can also obtain the corresponding mask of this whole-brain template. Perhaps, this could be used directly...

Reply:
Sorry for the lag. I think we should indeed use the mask as you suggested. This isn't really my area of expertise, but I would do something like:

```python
from nilearn.datasets import MNI152_FILE_PATH
from nilearn.masking import apply_mask, unmask
from nilearn.plotting import plot_img
template = load_img("./mni_icbm152_t1_tal_nlin_sym_09a.nii")
plot_img(template)
mask = load_img("./mni_icbm152_t1_tal_nlin_sym_09a_mask.nii")
niimg = unmask(apply_mask(template, mask), mask)
plot_img(niimg) # New
plot_img(MNI152_FILE_PATH) # Current
```
![mni](https://user-images.githubusercontent.com/2639645/116053640-34165880-a67b-11eb-9049-0fcb16ac6a1f.png)

Reply:
Thank you very much for the tip. I just pushed the new 2mm-resolution whole-brain mask and changed the MNI152_FILE_PATH. All masks, i.e. whole-brain, gm and wm, have now the same shape: (197, 233, 189). Yet, the scale of the whole-brain map is still different from the remaining ones. Shall this be changed, too?
![Figure_1](https://user-images.githubusercontent.com/10830228/116256283-58edf700-a773-11eb-8b1c-578fb1a60ad8.png)
![Figure_2](https://user-images.githubusercontent.com/10830228/116256301-5b505100-a773-11eb-9c9a-15917bd1da24.png)
![Figure_3](https://user-images.githubusercontent.com/10830228/116256317-5e4b4180-a773-11eb-94c0-21188f2dc6b4.png)

Reply:
Yes, I think we should change the scale too.

Reply:
Shall I divide the values of the whole-brain template by 100?

Reply:
@alpinho , maybe something like that?:

```python
import numpy as np
from nilearn.datasets import MNI152_FILE_PATH
from nilearn.image import get_data, new_img_like, load_img
ref = load_img(MNI152_FILE_PATH)
new_data = get_data(ref)
new_data /= np.max(new_data)
new = new_img_like(ref, new_data)
```

Reply:
This is what the new whole-brain template I just committed look like, which seems to be the desired outcome.
![Figure_4](https://user-images.githubusercontent.com/10830228/116409518-353eb580-a834-11eb-90f0-bcd876231f69.png)